lib/ansible/modules/system/user.py

```diff
@@ -2609,6 +2629,10 @@ def main():
                 module.fail_json(name=user.name, msg=err, rc=rc)
             result['force'] = user.force
             result['remove'] = user.remove
+            result['diff'] = {
+                'before': 'user exists\n',
```

Comment:
Return the actual username here:
```suggestion
                'before': '{0} exists\n'.format(user.name),
```